exercises/concept/valentines-day/.docs/hints.md

```diff
@@ -0,0 +1,29 @@
+# Hints
+
+## 1. Define the approval
+
+- [This page][ADT] shows how to define an algebraic data type.
+
+## 2. Define the cuisine
+
+- [This page][ADT] shows how to define an algebraic data type.
```

Comment:
because of https://www.w3.org/QA/Tips/noClickHere we're not allowed to use "This page"